### PR TITLE
example: add huge tasks example

### PR DESCRIPTION
**Pull request by tianweidut**
## Modules
- [ ] UI
- [ ] Controller
- [ ] Agent
- [ ] Client
- [ ] Python-SDK
- [x] Others

## Checklist
- [ ] run code format and lint check
- [ ] add unit test
- [ ] add necessary doc

### CODE REVIEW

File: example/huge-tasks/dataset.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+from starwhale.utils.debug import console
+
+try:
+    from .utils import random_text
+except ImportError:
+    from utils import random_text
+
+
+def build():
+    with dataset("huge-tasks-random-text-100w") as ds:
```
Comment (jialeicui) [inline example/huge-tasks/dataset.py:11]: better to use `1000k`

Comment (tianweidut) [reply example/huge-tasks/dataset.py:11]: updated

File: example/huge-tasks/dataset.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+try:
+    from .utils import random_text
+except ImportError:
+    from utils import random_text
+
+
+def build():
+    with dataset("huge-tasks-random-text-1m") as ds:
+        console.print(f"start to build dataset: {ds.uri.name}")
+        for i in range(0, 1000 * 1000 - 1):
```
Comment (goldenxinxing) [inline example/huge-tasks/dataset.py:13]: The rows number of range(0, 999999) is 999999, remove the '-1'?

Comment (tianweidut) [reply example/huge-tasks/dataset.py:13]: Only wants 999999 rows.

range(0, 100) --> 100 rows
range(0, 100-1) --> 99 rows

**Review by goldenxinxing — APPROVED** [review]
LGTM

**Pull request merged**